### DISCUSSION TITLE
Github Student developer pack

### DISCUSSION THREAD

**Discussion by goelarnav12** [community/community#128873] [GitHub Education]
### Hi everyone,

I have been approved on Jan 20 ,2024 for github student developer pack , I still havent been able to use the benefits , what can I do?

**Comment by davevad93** [comment]
Hi @goelarnav12 

A month ago I recall a discussion similar to yours where a user claimed that his benefits became available 165 days after the approval (almost 6 months). I don't know the exact criteria for the waiting time, you did well asking here in the community since someone from GitHub Education staff could fix it for you. Anyway I suggest you to also contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your situation.

**Reply by goelarnav12** [reply]
thanks , will do
